app/components/embed/footer_component.html.erb

```diff
@@ -12,6 +12,7 @@
 
     <% if viewer.show_download? %>
       <button
+        id='sul-embed-download-panel-toggle'
         class='sul-embed-footer-tool btn btn-toolbar sul-i-download-3'
```

Comment:
disambiguate this element from the download all link (which shares classes), helpful for testing